include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge.h

```diff
@@ -499,19 +499,14 @@ __get_starting_size_limit_for_large_submitter<int>()
     return 16 * 1'048'576; // 16 MB
 }
 
-template <typename _OutSizeLimit = std::false_type, typename _ExecutionPolicy, typename _Range1, typename _Range2,
+template <typename _CustomName, typename _OutSizeLimit = std::false_type, typename _Range1, typename _Range2,
           typename _Range3, typename _Compare, typename _Proj1 = oneapi::dpl::identity,
           typename _Proj2 = oneapi::dpl::identity>
 __future<sycl::event, std::shared_ptr<__result_and_scratch_storage_base>>
-__parallel_merge(oneapi::dpl::__internal::__device_backend_tag, _ExecutionPolicy&& __exec, _Range1&& __rng1,
-                 _Range2&& __rng2, _Range3&& __rng3, _Compare __comp, _Proj1 __proj1 = {}, _Proj2 __proj2 = {})
+__parallel_merge_impl(sycl::queue& __q, _Range1&& __rng1, _Range2&& __rng2, _Range3&& __rng3, _Compare __comp,
```

Comment:
We still have about 22 places where we have in params `oneapi::dpl::__internal::__device_backend_tag, _ExecutionPolicy&& __exec`
So my question - why we made some cleaning only for subset of functions in this PR which is for the new functional? May be better do not touch this in the current PR and make cleanup of all these places in some separate PR with `2022.11` milestone?

Reply:
ok